src/Components/PoiInformations.js

```diff
@@ -15,21 +15,29 @@ const mapStateToProps = state => ({
 const PoiInformation = ({
   dispatch, specificPoiInfos, InformationPoiInfos, geolocCoordonnees, handleKeyPress,
 }) => (
+
+
   <div>
     <div
       className={InformationPoiInfos ? 'informationPageTop' : 'informationPageBottom'}
-      onClick={() => dispatch({ type: 'TRANSITION_POI_INFOS', InformationPoiInfos: !InformationPoiInfos })}
-      role="button"
-      tabIndex="0"
-      onKeyPress={handleKeyPress}
     >
       <Close
         className="closePoiInformation"
         type="button"
         onClick={() => dispatch({ type: 'CLOSE_POI_INFOS', specificPoiInfos: [] })}
       />
 
-      <p className="poiName">{specificPoiInfos.name}</p>
+
+      <div
+        className="poiName"
+        onClick={() => dispatch({ type: 'TRANSITION_POI_INFOS', InformationPoiInfos: !InformationPoiInfos })}
```

Comment:
The change should be done in the reducer, not the action

Reply:
ok